### PR TITLE
Investigate 4559 v2

### PR DESCRIPTION
**Pull request by coolcorexix**
fixing bug mentioned in #4559 - `updateId` not properly updated

please help me review and instruct me further to complete this patch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "clean": "polkadot-dev-clean-build",
     "docs:metadata": "polkadot-types-internal-metadata",
     "lint": "polkadot-dev-run-lint",
+    "polkadot-dev-copy-to": "polkadot-dev-copy-to",
```
Comment (jacogr) [inline package.json:28]: This should not be needed?

**Review by jacogr — COMMENTED** [review]
Thank you! These PRs are the best. 

Will give it a good look in the morning, trying to take off tonight :)

**Comment by coolcorexix** [comment]
I have touched up the PR, please check and tell me what else is needed

**Review by jacogr — APPROVED** [review]
Thank you! Really appreciate it.

**Pull request merged**